src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/VideoAppointment.kt

```diff
@@ -47,6 +47,10 @@ data class VideoAppointment(
 interface AppointmentSlot {
   val prisonLocationId: UUID
 
+  val prisonerNumber: String
+
+  val appointmentDate: LocalDate
+
   val startTime: LocalTime
```

Comment:
Might be useful to add appointment type code to the comparison values? It narrows the comparison down further.